1-js/04-object-basics/04-object-methods/article.md

```diff
@@ -50,59 +50,59 @@ let user = {
 *!*
 // first, declare
 function sayHi() {
-  alert("Hello!");
+  alert("Salut!");
 };
 
 // then add as a method
 user.sayHi = sayHi;
 */!*
 
-user.sayHi(); // Hello!
+user.sayHi(); // Salut!
 ```
 
-```smart header="Object-oriented programming"
-When we write our code using objects to represent entities, that's called an [object-oriented programming](https://en.wikipedia.org/wiki/Object-oriented_programming), in short: "OOP".
+```smart header="Programare orientată pe obiect"
+Când scriem cod folosind obiecte pentru a reprezenta entități, asta se numește [programare orientată pe obiect (object-oriented programming)](https://en.wikipedia.org/wiki/Object-oriented_programming), pe scurt: "OOP".
 
-OOP is a big thing, an interesting science of its own. How to choose the right entities? How to organize the interaction between them? That's architecture, and there are great books on that topic, like "Design Patterns: Elements of Reusable Object-Oriented Software" by E.Gamma, R.Helm, R.Johnson, J.Vissides or "Object-Oriented Analysis and Design with Applications" by G.Booch, and more.
+OOP este un lucru mare, o știință interesantă în sine. Cum să alegi entitățile potrivite? Cum să organizezi interacțiunea dintre ele? Aceasta este arhitectură, și există cărți interesante pe această temă, precum "Design Patterns: Elements of Reusable Object-Oriented Software" de E.Gamma, R.Helm, R.Johnson, J.Vissides sau "Object-Oriented Analysis and Design with Applications" de G.Booch, și altele.
 ```
-### Method shorthand
+### Metoda scurtă
 
-There exists a shorter syntax for methods in an object literal:
+Există o sintaxă mai scurtă pentru metode într-un obiect literal:
 
 ```js
-// these objects do the same
+// aceste obiecte realizează același lucru
 
 user = {
   sayHi: function() {
-    alert("Hello");
+    alert("Salut");
   }
 };
 
-// method shorthand looks better, right?
+// Metoda scurtă arată mai bine, nu-i așa?
 user = {
 *!*
-  sayHi() { // same as "sayHi: function()"
+  sayHi() { // la fel ca "sayHi: function()"
 */!*
-    alert("Hello");
+    alert("Salut");
   }
 };
 ```
 
-As demonstrated, we can omit `"function"` and just write `sayHi()`.
+După cum s-a demonstrat, putem omite `"function"` și scriem doar `sayHi()`.
 
-To tell the truth, the notations are not fully identical. There are subtle differences related to object inheritance (to be covered later), but for now they do not matter. In almost all cases the shorter syntax is preferred.
+Pentru a spune adevărul, notațiile nu sunt complet identice. Există diferențe subtile legate de moștenirea obiectelor (care vor fi acoperite ulterior), dar deocamdată nu contează. În aproape toate cazurile, se preferă sintaxa mai scurtă.
```

Comment:
Expresia 'pentru a spune adevarul' nu suna tocmai ok. Poate ceva de genul ca 'in realitate' ar merge mai bine aici